packages/cra-template-apps-starter-kit/template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "package": {
+    "dependencies": {
+      "@datadog/ui-extensions-react": "0.30.1",
+      "@datadog/ui-extensions-sdk": "0.30.1",
+      "milligram": "^1.4.1",
+      "react": "^17.0.2",
+      "react-dom": "^17.0.2",
```

Comment:
The latest version of `create-react-app` will use React v18 if we don't specify a version here, but since the `@types/react` and `@types/react-dom` packages haven't been updated yet I don't think we can upgrade the `peerDependency` `react` in [`@datadog/ui-extensions-react](https://github.com/DataDog/apps/blob/master/packages/ui-extensions-react/package.json#L41).

Reply:
This is no longer the case.